packages/zero-next-plugin/src/index.ts

```diff
@@ -81,3 +82,5 @@ export function withZeroPlugin(nextConfig: NextConfig, zeroConfig: ZeroPluginCon
     webpack,
   };
 }
+
+export { extendTheme };
```

Comment:
Why do we need to reexport it from the plugin? I am afraid that it might be confusing for users especially with auto import/autocomplete in IDE.

Reply:
It'll reduce the mental overhead for users trying to import this.
Ideally, they should only think of `@mui/zero-runtime` imports to be imported only in their source code, not in the config files outside the source.

Reply:
Got it. What's your thought @mnajdova ?